spec.html

```diff
@@ -7535,6 +7534,17 @@ <h1>SetFunctionName ( _F_, _name_ [ , _prefix_ ] )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-setfunctionlength" aoid="SetFunctionLength">
+      <h1>SetFunctionLength ( _F_, _length_ )</h1>
+      <p>The abstract operation SetFunctionLength requires a Function argument _F_, and a Number argument _length_. This operation adds a `length` property to _F_ by performing the following steps:</p>
+      <emu-alg>
+        1. Assert: _F_ is an extensible object that does not have a `length` own property.
+        1. Assert: Type(_length_) is Number.
+        1. Assert: ToLength(_length_) is equal to _length_.
```

Comment:
ToLength is not correct here, because if called from `Function.prototype.bind` `length` can exceed 2^53-1. If a bounds check is wanted here, something like
  1. Assert: _length_ &ge; 0 and ToInteger(_length_) is equal to _length_.

is necessary.